rewrite-java-11/src/main/java/org/openrewrite/java/Java11ParserVisitor.java

```diff
@@ -1655,13 +1655,13 @@ private JavaType type(@Nullable com.sun.tools.javac.code.Type type, List<Symbol>
                     ClassType classType = (ClassType) type;
                     ClassType symType = (ClassType) sym.type;
 
-                    List<JavaType> typeParameters;
+                    List<JavaType.FullyQualified> typeParameters;
                     if (classType.typarams_field == null) {
                         typeParameters = emptyList();
                     } else {
                         typeParameters = new ArrayList<>();
                         for (com.sun.tools.javac.code.Type tParam : classType.typarams_field) {
-                            JavaType javaType = type(tParam, stackWithSym, true);
+                            JavaType.FullyQualified javaType = TypeUtils.asFullyQualified(type(tParam, stackWithSym, true));
```

Comment:
I think this could be a `? extends J` which is not a `FullyQualified`.

Reply:
If you agree, just keep the weaker type.

Reply:
oh, you are right, I forgot about `List<String[]>`, perfectly valid. I will change that.